validator/sawtooth_validator/consensus/handlers.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         self._response_class = response_class
         self._response_type = response_type
 
-    def handle_request(self, request, response):
+    def handle_request(self, request, connection_id):
```

Comment:
Looks like you replaced `response` but it is still required by other implementors, and at other call sites.